main.js

```diff
@@ -3,14 +3,22 @@ require("babel-polyfill")
 
 window.engine = {}
 const app = new Tram({
+  defaultRoute: '/',
   webEngine: window.engine
 })
 
-app.addRoute('/', require('./pages/home'))
-app.addRoute('/404', require('./pages/404'))
+const route = Tram.route()
 
-app.addRoute('/privacy', require('./pages/privacy'))
-app.addRoute('/opensource', require('./pages/opensource'))
-app.addActions({ update: require('./actions/update')})
+app.addRoute('/', require('./pages/wrapper'), [
+  route('/', require('./pages/home')),
+  route('/:color', require('./pages/home')),
+  route('/privacy', require('./pages/privacy')),
+  route('/opensource', require('./pages/opensource')),
+])
```

Comment:
using subroutes and route parameters.

the `/pages/wrapper` contains header and footer content that you would want on all pages. The subroutes are injected in the wrapper depending on the route.

`/:color` allows us to have routes like `/red`, `/orange`, `/green`, or `/blue`, and load just a single trains content on the page